core/templates/domain/admin/beam-job.model.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Domain object for Apache Beam jobs.
+ */
+
+export interface BeamJobBackendDict {
+  'name': string;
+  'parameter_names': string[];
+}
+
+export class BeamJob {
```

Comment:
What is the difference between `BeamJob` and `BeamJobRun`?

Reply:
`BeamJob` is the "definition" of a job and describes it, `BeamJobRun` is an "instance" of a job that is currently running or has run in the past. I've added comments explaining this, WDYT?

Reply:
LGTM!